04-subcharts/README.md

```diff
@@ -1,48 +1,71 @@
+stable/sensu                      0.2.3         0.28          Sensu monitoring framework backed by the Redis transport  
 ```
 
-See how we already have a chart that adds `redis` support? Rather than create our own, let's use that. (Note: If you are feeling extra ambitious, you can write your own Redis chart. But it will take about 45 minutes.)
+See how we already have a chart that adds `redis` support? Rather than create
+our own, let's use that. (Note: If you are feeling extra ambitious, you can
+write your own Redis chart. But it will take about 45 minutes.)
 
 ## Subcharts and `requirements.yaml`
 
-In Helm, a _subchart_ is a regular Helm chart that is a dependency of your chart. By adding it to a special file called `requirements.yaml`, you can let Helm manage that chart for you.
+In Helm, a _subchart_ is a regular Helm chart that is a dependency of your
+chart. By adding it to a special file called `requirements.yaml`, you can let
+Helm manage that chart for you.
 
-Let's create a `requirements.yaml` file next to the `Chart.yaml` in our chart. Here's what belongs inside:
+Let's create a `requirements.yaml` file next to the `Chart.yaml` in our chart.
+Here's what belongs inside:
 
 ```yaml
 dependencies:
   - name: redis        # from search results above
-    version: 3.7.6     # Also from the search results above
-    repository: https://kubernetes-charts.storage.googleapis.com
+    version: 7.1.0    # Also from the search results above
+    repository: "@stable" # Using the @ symbol allows you to use the name of a repository instead of the URL
 ```
 
-Note that the `repository` URL comes from running `helm repo list` and copying the URL for the `stable` repo, which is where the Redis chart lives.
-
 ### Add PostgreSQL on your own
 
-Next, run `helm search postgresql` and repeat the above for Postgres. By the end of that step, you should have two items in your `dependencies` array in `requirements.yaml`
+Next, run `helm search postgresql` and repeat the above for Postgres. By the end
+of that step, you should have two items in your `dependencies` array in
+`requirements.yaml`
 
 ## Download the Dependencies
 
-Run `helm dep up` to have Helm fetch those dependencies for you. When completed, you should be able to see two new directories inside of your chart's `charts` subdirectory.
+Run `helm dep up` to have Helm fetch those dependencies for you. When completed,
+you should be able to see two new directories inside of your chart's `charts`
+subdirectory.
 
-## Upgrade Your Release
+```console
+$ helm dep up
+Hang tight while we grab the latest from your chart repositories...
+...Unable to get an update from the "local" chart repository (http://127.0.0.1:8879/charts):
+  Get http://127.0.0.1:8879/charts/index.yaml: dial tcp 127.0.0.1:8879: connect: connection refused
+...Successfully got an update from the "incubator" chart repository
```

Comment:
Will other new users have the incubator chart repository installed? I don't have it by default.

Reply:
Oh no they won't. Good catch